packages/named-locks/src/index.ts

```diff
@@ -65,14 +74,17 @@ interface LockOptions {
  */
 
 export const pool = new PostgresPool();
+let renewIntervalMs = 60_000;
 
 /**
  * Initializes a new {@link PostgresPool} that will be used to acquire named locks.
  */
 export async function init(
   pgConfig: PoolConfig,
-  idleErrorHandler: (error: Error, client: PoolClient) => void
+  idleErrorHandler: (error: Error, client: PoolClient) => void,
+  namedLocksConfig: NamedLocksConfig = {}
 ) {
+  renewIntervalMs = namedLocksConfig.renewIntervalMs ?? renewIntervalMs;
```

Comment:
```suggestion
  renewIntervalMs ??= namedLocksConfig.renewIntervalMs;
```

Reply:
I don't think this operator does what we want. This will assign `renewIntervalMs` only if `renewIntervalMs` is `null`/`undefined`, but that will never be true because we always initialize it with a default value.